lib/pbench/server/api/resources/query_apis/datasets/datasets_mappings.py

```diff
@@ -81,25 +81,28 @@ def __init__(self, config: PbenchServerConfig, logger: Logger):
             ),
         )
 
-    def _get(self, dataset_view: AnyStr, request: Request) -> Dict[AnyStr, Any]:
+    def _get(self, json_data: JSON, request: Request) -> Response:
         """
-        Return mapping properties of the document specified by the index key
-        in the URI (supported index keys are defined in RunIdBase.ES_INTERNAL_INDEX_NAMES).
+        Return mapping properties of the document specified by the dataset view
+        in the URI (supported dataset views are defined in
```

Comment:
I suggest adding a word or two here, e.g., "by the dataset view _parameter_ in the URI" or " by the dataset view _specified_ in the URI _parameter_".